sdk/pinocchio/src/lib.rs

```diff
@@ -235,6 +235,7 @@ pub mod program {
 }
 pub mod program_error;
 pub mod pubkey;
+pub mod sanitize_error;
```

Comment:
There was only one place using this error type, so I think we can just drop it and use `ProgramError` instead.